cmake/fw_zip.cmake

```diff
@@ -18,13 +18,35 @@ function(generate_dfu_zip)
     message(FATAL_ERROR "Missing required param")
   endif()
 
+  find_package(Git QUIET)
+  if(GIT_FOUND)
+    execute_process(
```

Comment:
Note that this exact operation is also done in nrf/lib/modem_info/CMakeLists.txt, so perhaps this should be done just once instead in nrf/cmake/version.cmake or another general CMake file, and reused both places?

Reply:
I agree -- some sort of generic CMake function for running git describe and saving the results sounds reasonable. @tejlmand?